iconrpcserver/utils/json_rpc.py

```diff
@@ -93,8 +90,12 @@ def process_response(self, response, log_extra=None, log_format=None):
                     f"relay_target[{relay_target}], "
                     f"version[{version}], "
                     f"method[{method_name}]")
-        result = await CustomAiohttpClient(session, relay_uri).request(method_name, message)
+        print("type: ", CustomAiohttpClient.request)
+        client = CustomAiohttpClient(session, relay_uri)
+        print("cli: ", client)
+        result = await client.request(method_name, message)
         Logger.debug(f"relay_tx_request result[{result}]")
+        print(f"relay_tx_request result[{result}]")
```

Comment:
Is this draft?
I think this lines need to update.

Reply:
I made a mistake.
I will remove debug lines.

Reply:
Sorry to be late!